website/docs/cloud-docs/api-docs/run-tasks/run-task-stages-and-results.mdx

```diff
@@ -20,13 +20,14 @@ page_title: Run Task Stages and Results - API Docs - Terraform Cloud and Terrafo
 
 Terraform Cloud uses run task stages and run task results to track [run task](/cloud-docs/workspaces/settings/run-tasks) execution.
 
-When Terraform Cloud creates a [run][], it reads the run tasks associated to the workspace. Each run task in the workspace is configured to begin during a specific [run stage](/cloud-docs/run/states). Terraform Cloud creates a run task stage object for each run stage that will trigger run tasks. Currently, you can only configure run tasks during the [Post-Plan Stage](/cloud-docs/run/states#3-the-post-plan-stage), so Terraform Cloud produces one `post-plan` run task stage for each run.
+When Terraform Cloud creates a [run][], it reads the run tasks associated to the workspace. Each run task in the workspace is configured to begin during a specific [run stage](/cloud-docs/run/states). Terraform Cloud creates a run task stage object for each run stage that will trigger run tasks. You can configure run tasks during the [Pre-Plan Stage](/cloud-docs/run/states#3-the-pre-plan-stage) and [Post-Plan Stage](/cloud-docs/run/states#5-the-post-plan-stage).
+
+-> **Note:** Pre-plan run tasks are in beta. 
```

Comment:
Do we need to provide any further detail as to what it means to be in "Beta"?

cc/ @laurapacilio I wonder if we should have a page dedicated to `Beta` and what it means, it would allow us to link to it from notes such as this.

Reply:
Hi - sorry I missed this in all my other GitHub noise!  I love that idea. I think we should bring it up with the product team, as I know it's definitely not content I could draft myself. But I would love to have something like that to better set expectations for users!

Reply:
@cam-stitt   Do we need track this in on our board? I don't think we do.

Reply:
No we don't!